modules/prebidServerBidAdapter/index.js

```diff
@@ -637,6 +637,11 @@ const OPEN_RTB_PROTOCOL = {
       utils.deepSetValue(request, 'user.ext.eids', bidUserIdAsEids);
     }
 
+    const eidPermissions = utils.deepAccess(bidRequests, '0.bids.0.eidPermissions');
```

Comment:
I may be misunderstanding this, but it's not allowed to pass the eidPermissions directly to bidders.

i.e if this implies that the eidPermissions block is passed around on the bidRequest object, that would let client-side adapters know where they stand.

Reply:
Ok, I`ll change it